### DISCUSSION TITLE
Aqara P2 -- starts interview and fails every time.

### DISCUSSION THREAD

**Discussion by Ltek** [Koenkk/zigbee2mqtt#22533] [Devices]
anyone seen this issue? its a bear to get Z2M to startthe interview (about 1 out of every 5 tries) but every time it fails to interview... no debug info saying anything either. I have other, older, Aqara motions running just fine (they were also a nightmare to get paired but they didnt fail the interviews once started -- and I pressed button to keep them awake)

**Comment by francisp2** [comment]
is the Aqara p2 not matter over thread instead of Zigbee ?

**Reply by Ltek** [reply]
its zigbee.. I got it added but took several tries.
